resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -711,6 +719,27 @@ public Builder ignoreException(Predicate<Throwable> predicate) {
             return this;
         }
 
+        /**
+         * Configures whether unknown exceptions should be ignored by the circuit breaker.
+         *
+         * <p>
+         *     If set to {@code true}, the circuit breaker will ignore unknown exceptions listed in the
+         *     'ignoreExceptions' field during application startup, preventing the application from failing to start.
+         * </p>
+         *
+         * <p>
+         *     The default value is {@code false}, meaning unknown exceptions will trigger a {@code ClassNotFoundException}
+         *     during startup.
+         * </p>
+         *
+         * @param ignoreUnknownExceptions {@code true} to ignore unknown exceptions during startup.
+         * @return CircuitBreakerConfig.Builder
+         */
+        public final Builder ignoreUnknownExceptions(@Nullable boolean ignoreUnknownExceptions) {
```

Comment:
Let's use ignoreClassBindingExceptions.
It does not configure that the CircuitBreaker ignores unknown exceptions. This is confusing. 
IgnoreExceptions configures which exceptions are ignore at runtime to secure a HTTP.
`ignoreClassBindingExceptions` is a setting which should not effect the runtime behavior of the CircuitBreaker, but instead make sure that Classes which are not found are ignored and don't lead to a failure at startup phase..

Reply:
Actually I think we should not add this config parameter to the CircuitBreakerConfig.
Since it does not change the behavior of the CircuitBreaker at all.
If the config parameter is only useful in frameworks such as Spring Boot, we should move the config property to the CommonCircuitBreakerConfigurationProperties